samples/backups-get-database-operations.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2020 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+async function getDatabaseOperations(instanceId, projectId) {
+  // [START spanner_list_database_operations]
+  // Imports the Google Cloud client library
+  const {Spanner} = require('@google-cloud/spanner');
+  const {google} = require('../protos/protos');
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const projectId = 'my-project-id';
+  // const instanceId = 'my-instance';
+
+  // Creates a client
+  const spanner = new Spanner({
+    projectId: projectId,
+  });
+
+  // Gets a reference to a Cloud Spanner instance
+  const instance = spanner.instance(instanceId);
+
+  // List database operations
+  try {
+    const [databaseOperations] = await instance.getDatabaseOperations({
+      filter:
+        '(metadata.@type:type.googleapis.com/google.spanner.admin.database.v1.OptimizeRestoredDatabaseMetadata)',
+    });
+    console.log('Optimize Database Operations:');
+    databaseOperations.forEach(databaseOperation => {
+      const metadata = google.spanner.admin.database.v1.OptimizeRestoredDatabaseMetadata.decode(
+        databaseOperation.metadata.value
+      );
+      console.log(
+        `Database ${metadata.name} restored from backup is ` +
+          `${metadata.progress.progress_percent}% optimized.`
```

Comment:
@mayurkale22 the `metadata.progress.progress_percent` field is undefined even though `console.log(metadata)` shows that the field is there. This is likely a type issue but I'm not sure how to debug it.

The type is defined [here](https://github.com/googleapis/nodejs-spanner/blob/e9efc3f5d19f2991fcd6db5d861850f682f14571/proto/spanner_database_admin.d.ts#L2360).

Reply:
I think it should be like `metadata.progress.progressPercent`.

Reply:
I tried locally, it is working for me.

Reply:
Doh! My bad. Thanks for spotting that.